prime-angular-frontend/src/app/core/resources/site-resource.service.ts

```diff
@@ -411,6 +411,26 @@ export class SiteResource {
       );
   }
 
+  public flagSite(siteId: number, flagged: boolean): Observable<Site> {
```

Comment:
So this just seems an odd way to do this, compared to just having a PUT that you can pass a boolean to. But i see you just cloned ```updateEnrolleeAlwaysManual```

Im curios to have a conversation with the team about this pattern. But if we do keep it this way can we maybe call this method. updateSiteFlag, since flagSite would lead me to believe this sets the site flag to true.